tests/e2e/e2e_util_test.go

```diff
@@ -310,64 +246,104 @@ func (s *IntegrationTestSuite) execGovSubmitProposal(c *chain, valIdx int, endpo
 		"gov",
 		"submit-proposal",
 		govProposalPath,
-		fmt.Sprintf("--%s=%s", flags.FlagFrom, submitterAddr),
 		fmt.Sprintf("--%s=%s", flags.FlagGasPrices, fees),
-		fmt.Sprintf("--%s=%s", flags.FlagChainID, c.id),
-		"--keyring-backend=test",
-		"--output=json",
-		"-y",
 	}
 
-	s.executeGaiaTxCommand(ctx, c, gaiaCommand, valIdx, endpoint)
+	s.executeGaiaTxCommand(ctx, c, gaiaCommand, valIdx, submitterAddr)
 	s.T().Logf("Successfully submitted proposal %s", govProposalPath)
 }
 
-func (s *IntegrationTestSuite) executeGaiaTxCommand(ctx context.Context, c *chain, gaiaCommand []string, valIdx int, endpoint string) {
+func (s *IntegrationTestSuite) executeGaiaTxCommand(ctx context.Context, c *chain, gaiaCommand []string, valIdx int, from string) {
 	var (
 		outBuf bytes.Buffer
 		errBuf bytes.Buffer
 		txResp sdk.TxResponse
+		// accountResp authtypes.QueryAccountResponse
+		err error
 	)
+	endpoint := fmt.Sprintf("http://%s", s.valResources[c.id][0].GetHostPort("1317/tcp"))
 
-	s.Require().Eventually(
-		func() bool {
-			time.Sleep(3 * time.Second)
-			exec, err := s.dkrPool.Client.CreateExec(docker.CreateExecOptions{
-				Context:      ctx,
-				AttachStdout: true,
-				AttachStderr: true,
-				Container:    s.valResources[c.id][valIdx].Container.ID,
-				User:         "root",
-				Cmd:          gaiaCommand,
-			})
-			s.Require().NoError(err)
+	res, err := s.queryAuthAccount(endpoint, from)
+	s.Require().NoError(err)
 
-			err = s.dkrPool.Client.StartExec(exec.ID, docker.StartExecOptions{
-				Context:      ctx,
-				Detach:       false,
-				OutputStream: &outBuf,
-				ErrorStream:  &errBuf,
-			})
-			s.Require().NoError(err)
+	var acc authtypes.AccountI
+	err = cdc.UnpackAny(res.Account, &acc)
+	s.Require().NoError(err)
+	sequence := acc.GetSequence()
+
+	exec, err := s.dkrPool.Client.CreateExec(docker.CreateExecOptions{
+		Context:      ctx,
+		AttachStdout: true,
+		AttachStderr: true,
+		Container:    s.valResources[c.id][valIdx].Container.ID,
+		User:         "root",
+		Cmd: append(gaiaCommand,
+			fmt.Sprintf("--%s=%s", flags.FlagFrom, from),
+			fmt.Sprintf("--%s=%d", flags.FlagSequence, sequence),
```

Comment:
sequence+1 ?

Reply:
nope it's sequence, i thought the same thing at first